Cython/Compiler/Symtab.py

```diff
@@ -987,6 +1012,11 @@ class FakeOperand(Node):
         operands = [FakeOperand(pos, type=type) for type in types]
         return self.lookup_operator(operator, operands)
 
+    def _lookup_python_mangled_warning(self, pos, name):
+        warning(pos, "Global name %s matched from within class scope "
+                            "in contradiction to to Python 'class private name' rules. "
+                            "This may change in a future release." % name, 1)
```

Comment:
This method doesn't do a lookup, it emits a warning. The name should reflect that.